src/languages/asciidoc.js

```diff
@@ -18,8 +18,8 @@ export default function(hljs) {
         '^/{4,}\\n',
         '\\n/{4,}$',
         // can also be done as...
-        //'^/{4,}$',
-        //'^/{4,}$',
+        // '^/{4,}$',
+        // '^/{4,}$',
```

Comment:
Is this related to the unconstrained change

Reply:
Not really, eslint was unhappy with the missing space after `//` but not mandatory I can revert this change if you want.

Reply:
Oh comments, duh... I missed that at just a glance... no this is fine as it's minor.